src/mon/PGMap.cc

```diff
@@ -1514,7 +1514,12 @@ void PGMap::update_delta(CephContext *cct,
   delta_t -= *last_ts;  // take the last timestamp we saw
   *last_ts = ts;        // @p ts becomes the last timestamp we saw
 
-  // calculate a delta, and average over the last 2 deltas.
+  // adjust delta_t, quick start if there is no update in a long period
+  if (result_ts_delta->is_zero()) {
```

Comment:
Why is this condition needed?  It seems like you could do the min unconditionally...